src/components/Input/input.css

```diff
@@ -0,0 +1,20 @@
+.input-box {
+  width: 95%;
```

Comment:
make it 100% and add 
  box-sizing: border-box;
  
  to keep it inside screen